WordPress/gutenberg
# Update media and video icons

## What?
Was fixing the core/media-text media-container-icon and noticed that the `media` and `video` icons are the same icons. The `media` icon is used when referencing the Media Library ([storybook](https://wordpress.github.io/gutenberg/?path=/story/icons-icon--library)), even though its technically a video icon:

<img width="269" alt="CleanShot 2023-03-31 at 19 24 30" src="https://user-images.githubusercontent.com/1813435/229249384-d15c243b-7c13-4679-8d3d-d420f497cb48.png">
<img width="236" alt="CleanShot 2023-03-31 at 19 24 59" src="https://user-images.githubusercontent.com/1813435/229249403-4f7e4611-4f0c-4d4b-9704-3a1f8eb042e9.png">

I have two proposed paths to take: 

1. Update the `media` icon to accurately reflect images and videos. Then tweak the places where the existing `media` icons are used, to reference the actual `video` icon (what this PR does). 

2. Update the `media` icon to accurately reflect images and videos. Keep the new `media` icon assigned to the Media Library menu items. The downside here is that it's not always the case that images can be replaced by videos (but perhaps it should be). Currently, the Image block does not allow for video file types. 

Another alternative would be to add a new icon to the library, but I don't feel it's necessary while this media/video icon discrepancy is in play. 

## Testing Instructions

1. Open a Post or Page. 
2. Insert a Media & Text block.
4. See new `media` icon.
5. Open the "Replace" popover within the block's toolbar. 
6. See no changes with the existing "Media Library" icon (uses the `video` icon). 

## Screenshots or screencast 

The custom media-container-icon in the Media & Text block in trunk. It doesn't fit the design language of all the other icons, and shouldn't be its own custom icon in the block anyhow:

<img width="767" alt="CleanShot 2023-03-31 at 19 38 47" src="https://user-images.githubusercontent.com/1813435/229250499-649b0e5f-6b2c-4a0d-a698-9058e69b0659.png">

The updated `media` icon, now in use within the Media & Text block placeholder state:

<img width="97" alt="CleanShot 2023-03-31 at 19 43 11" src="https://user-images.githubusercontent.com/1813435/229250894-e69c4e8d-d44c-4d23-86de-b62a01b085d1.png">

<img width="766" alt="CleanShot 2023-03-31 at 19 34 10" src="https://user-images.githubusercontent.com/1813435/229250218-e458071e-801f-40d1-beed-cb141db1a482.png">

The `video` icon, now in use within Media Library menu items. There is no visual change; just referencing the actual video icon.  

<img width="705" alt="CleanShot 2023-03-31 at 19 28 52" src="https://user-images.githubusercontent.com/1813435/229249771-8e1c7e8d-f981-4847-badf-50376990911f.png">
